src/consensus/pbft/libbyz/parameters.h

```diff
@@ -7,6 +7,13 @@
 
 #include <cstddef>
 
+// keypair parameters
```

Comment:
moved here so we can use `signature_size` in `pbftpreprepares.h`

Reply:
What do we use SHA512 for?

Reply:
https://github.com/microsoft/CCF/issues/671